widgetastic_manageiq/__init__.py

```diff
@@ -644,13 +644,18 @@ class SummaryTable(VanillaTable):
 
     Args:
         title: Title of the table (eg. ``Properties``)
+        locator: If the SummaryTable you are using has different BASE Locator, it can be passed here
     """
 
     BASELOC = './/table[./thead/tr/th[contains(@align, "left") and normalize-space(.)={}]]'
```

Comment:
Just changing the 'AND' operator to 'OR' might help here ?

 BASELOC = './/table[./thead/tr/th[contains(@align, "left") or normalize-space(.)={}]]'